bands/models.py

```diff
@@ -18,81 +18,106 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info_id = Column(UUID, ForeignKey('band_info.id'), nullable=True)
+    bio_id = Column(UUID, ForeignKey('band_bio.id'), nullable=True)
+    taxes_id = Column(UUID, ForeignKey('band_taxes.id'), nullable=True)
+    stageplot_id = Column(UUID, ForeignKey('band_stage_plot.id'), nullable=True)
+    panel_id = Column(UUID, ForeignKey('band_panel.id'), nullable=True)
+    merch_id = Column(UUID, ForeignKey('band_merch.id'), nullable=True)
+    charity_id = Column(UUID, ForeignKey('band_charity.id'), nullable=True)
```

Comment:
Ack, you're right - I thought I had screwed this up. I got confused because the examples in this repo were the reverse relationship. Do any of the relationship columns need special settings?